### PR TITLE
Document overall "neonatal conditions" cause model

### PR DESCRIPTION
**Pull request by aflaxman**

### CODE REVIEW

File: docs/source/models/causes/neonatal/index.rst [hunk 1]
```diff
@@ -29,45 +29,231 @@ the strategy for capturing the burden of the neonatal subcauses to calibrate to
...
+
+
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
+The excluded nonfatal portion of the burden is small (around 10% of the total burden).
+
+The evidence base for identifying the level 4 subcauses is not as solid as I would like.
+
+Preterm must be split into with and without respiratory distress syndrome (RDS) outside of GBD, since GBD does not distinguish preterms deaths with and without RDS.
```
Comment (pletale) [inline docs/source/models/causes/neonatal/index.rst:129]: Do we have a data strategy for RDS beyond using the preterm birth cause from GBD?

File: docs/source/models/causes/neonatal/index.rst [hunk 2]
```diff
@@ -29,45 +29,231 @@ the strategy for capturing the burden of the neonatal subcauses to calibrate to
...
+We are not modeling neonatal disorders dynamically as a finite state machine, but we can draw an directed 
+graph to represent the collapsed decision tree  
+representing this cause. Unlike a state machine representation, the values on the 
+transition arrows represent decision probabilities rather than rates per 
+unit time.
+
+
+.. graphviz::
+
+    digraph OL_decisions {
```
Comment (pletale) [inline docs/source/models/causes/neonatal/index.rst:143]:
```suggestion
    digraph nn_decisions {
```

File: docs/source/models/causes/neonatal/index.rst (hunk 1 above)
Comment (pletale) [reply docs/source/models/causes/neonatal/index.rst:129]: And will we be documenting this on the preterm birth subcause model page?

**Review by pletale — APPROVED** [review]

File: docs/source/models/causes/neonatal/index.rst (hunk 1 above)
Comment (NathanielBlairStahn) [reply docs/source/models/causes/neonatal/index.rst:129]: Good questions. In regards to the first one, in the [nanosim](https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/lbwsg_nanosim/2024_09_17_ai_ultrasound_pathway_nanosim.ipynb), I [used Table 4](https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/lbwsg_nanosim/preterm_cod.py) from [this paper](https://www.sciencedirect.com/science/article/pii/S2214109X19302207) (emailed to us by Chris Troeger) to estimate that about 85% of preterm birth deaths are due to RDS. We could use the same table to get percentages stratified by gestational age bins as well. However, this paper is from a single study in Ethiopia -- I don't know where to find more wide-ranging data.

File: docs/source/models/causes/neonatal/index.rst [hunk 3]
```diff
@@ -29,45 +29,231 @@ the strategy for capturing the burden of the neonatal subcauses to calibrate to
...
-Subcause definitions
-""""""""""""""""""""""""""""
+- All causes (c_294) [level 0]
+
+  - Communicable, maternal, neonatal, and nutritional diseases (c_295)
+
+    - Maternal and neonatal disorders (c_962)
+
+      - **Neonatal disorders (c_380)** [level 3]
+
```
[the remark this answers is not in the document]
Comment (aflaxman) [reply docs/source/models/causes/neonatal/index.rst:40]: I do not find these lists useful.  Do you think the engineers need it?  Maybe it is needed for implementing and checking the YLD part of a model (which I am excluding in this model).

File: docs/source/models/causes/neonatal/preterm_birth.rst [hunk 4]
```diff
@@ -1,47 +1,221 @@
...
+
+    * - State
+      - Definition
+    * - live birth
+      - The parent simulant has given birth to a live child simulant (which
+        is determined in the
+        intrapartum step of the :ref:`pregnancy model
+        <other_models_pregnancy_closed_cohort_mncnh>`)
+    * - neonate survived
+      - The child simulant survived for the first 28 days of life
```
Comment (NathanielBlairStahn) [inline docs/source/models/causes/neonatal/preterm_birth.rst:130]: The node name should be changed to "neonate did not die of preterm", and the definition should be changed accordingly.

File: docs/source/models/causes/neonatal/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (aflaxman) [reply docs/source/models/causes/neonatal/index.rst:40]: I'll add in a future PR; @pletale : can you tell me where to find this info?

**Pull request merged**